.github/workflows/ci.yml

```diff
@@ -225,3 +231,40 @@ jobs:
         with:
           name: Test results (editmode)
           path: artifacts/test/editmode
+
+  android-smoke-test:
+    needs: [build]
+    name: Unity ${{ matrix.unity-version }} - macos-latest - Android ${{ matrix.api-level }} smoke tests 
+    runs-on: macos-latest
+    strategy:
+      matrix:
+         api-level: [21, 26, 29]
+         # 2021.1.26f1 is not being generated.
```

Comment:
We should only skip this version on Windows due o the PathTooLong issue (which is a Unity issue). Since it builds on macOS, we should have an artifact here to test